README.md

```diff
@@ -256,9 +256,53 @@ To configuring a pipeline:
 | [stage](https://docs.gitlab.com/ee/ci/yaml/#stage) | Defines a job stage (E.g., build) |
 | [variables](https://docs.gitlab.com/ee/ci/yaml/#variables) | Define job variables on a job level |
 
-### GitHub Pipelines
+## GitHub 
+
+### Set-up remote 
+
+* Rename current Gitlab remote
+  ```bash
+  $ git remote rename origin gitlab
+  ```
+
+* Update Git to add new remote to GitHub: 
+  ```bash
+  $ git remote add github git@github.com:arzukhanna/sudoku-python.git
+  ```
+
+* Force push main branch from GitLab to GitHub
+  ```bash
+  $ git checkout main
+  $ git push -fu github main
+  ```
+  
+* Push develop and feature branch from GitLab to GitHub
+  ```bash
+  $ git checkout develop
+  $ git push -u github develop
+  $ git checkout feature
+  $ git push -u github feature
+  ```
+
+* In GitHub, protect develop and main branches. These are long-lived and 
+  should be "protected". ( Settings > Branches)
+
+### Pipelines
+
+A pipeline is declared using a YAML file, [.github-actions.yml](.github-actions.yml) 
```

Comment:
This won't work as you have to specify full path. 
```text
[text](path/to/file)
```
Oh, and the name has changed.